mintlify/snippets/variables.mdx

```diff
@@ -1,3 +1,3 @@
 export const topLevelProductName = "Lightspark Grid";
-export const umaaasProductName = "UMA Connect";
+export const umaaasProductName = "UMA Payments";
```

Comment:
This doesn't seem right to me from a product naming standpoint. Every product here can send UMA Payments. The unique thing about UMAaaS or UMA Connect is that it's geared towards banks or regulated institutions who want to give their users addresses and send/receive P2P payments.